examples_tests/20.Megatexture/main.cpp

```diff
@@ -227,6 +225,8 @@ core::smart_refctd_ptr<asset::ICPUSpecializedShader> createModifiedFragShader(co
     return fsNew;
 }
 
+/* TODO: add it after mergin Dan PR
```

Comment:
TODO: add event system handling utilities from @sadiuk for catching keyboard input events in near future